projects/demo/demo.py

```diff
@@ -90,9 +95,9 @@ def main(args):
             dcc.Dropdown(
                 id="dataset_dropdown",
                 options=[
-                    {"label": f"{title}", "value": id_} for id_, title in DATASETS
+                    {"label": f"{title}", "value": id_} for id_, title in datasets
                 ],
-                value="subj",
+                value=datasets[0][0] if args.logfile else 'subj',
```

Comment:
Thanks for improving this demo and removing the hardcoded values. You can remove the if and always get the values from `datasets[0][0]`

Reply:
Thank you! I have removed the hard-coding with if.